tests/integration/components/bs-dropdown-test.js

```diff
@@ -365,6 +365,26 @@ module('Integration | Component | bs-dropdown', function (hooks) {
     assert.notStrictEqual(this.element.querySelector('.dropdown-menu').offsetParent, null);
   });
 
+  test('dropdown in nav disables dynamic positioning', async function (assert) {
+    await render(hbs`
+    <BsDropdown @inNav={{true}} as |dd|>
+      <dd.toggle>Dropdown</dd.toggle>
+      <dd.menu as |menu|>
+        <menu.item>
+          <menu.link-to @route="index">Home</menu.link-to>
+        </menu.item>
+      </dd.menu>
+    </BsDropdown>`);
+
+    await click('a.dropdown-toggle');
+
+    assert.equal(
```

Comment:
Please use `assert.strictEqual`. Please see this ESLint rule for details: https://github.com/platinumazure/eslint-plugin-qunit/blob/master/docs/rules/no-assert-equal.md

```suggestion
    assert.strictEqual(
```